src/Dfe.Spi.Common/Dfe.Spi.Common.Logging/Definitions/ILoggerWrapper.cs

```diff
@@ -13,10 +13,10 @@ public interface ILoggerWrapper
         /// Sets the context (i.e. custom dimension information) against the
         /// instance.
         /// </summary>
-        /// <param name="requestResponseBase">
-        /// An instance of <see cref="RequestResponseBase" />.
+        /// <param name="headerDictionary">
+        /// An instance of type <see cref="IHeaderDictionary" />.
         /// </param>
-        void SetContext(RequestResponseBase requestResponseBase);
+        void SetContext(IHeaderDictionary headerDictionary);
```

Comment:
As we have taken a dependency on `Microsoft.AspNetCore.Http` (which is fine), is it worth passing the `HttpRequest` in here? Just means could take more from the request at a later date without a signature change